pset-iterators.js

```diff
@@ -114,6 +127,12 @@ Expected Result Shape: An array of objects (with each object representing a pres
 console.log('Problem 4:')
 
 // Add your code below this line
+const presidentsBetween1850and1900 = presidents.filter((officeTime) => {
+  const officeResult = officeTime.took_office.split(" ");
```

Comment:
@cbrooks2019  seems like this works, however, you the split you used did not work as you expected since you passed an empty string `.split(" ")` instead of using `.split("-")` since the dates in the array data are structured like this: `"1797-03-04"`

In this instance, the  result is correct - take a look at the approached use in the solution for more details: https://github.com/jsd20191008/hw-02-iterators-solution/blob/master/pset-iterators.js#L124